blazor/file-manager/getting-started-with-server-app.md

```diff
@@ -196,10 +215,27 @@ app.MapControllers();.
 
 This will configure and map the controller in your Blazor App.
 
-To access the above File Operations, you need some model class files that have file operations methods. So, create `Models` folder in `server` part of the application and download the `PhysicalFileProvider.cs` and `Base` folder from the [this](https://github.com/SyncfusionExamples/ej2-aspcore-file-provider/tree/master/Models) link in the Models folder.
+## Create Web Server App
 
-Add your required files and folders under the `wwwroot\Files` directory.

```

Comment:
This line is removed but not added anywhere in new changes. Add it in adding controllers part. @Saravanan-sf4375